services/common/src/redux/slices/verifiableCredentialsSlice.ts

```diff
@@ -29,82 +29,89 @@ const initialState: VerifiableCredentialsState = {
   verifiableCredentialConnections: [],
 };
 
-const reducers = (create) => ({
```

Comment:
There were some typing errors here if reducers were defined outside of the createAppSlice and then passed in.  Defining them inside fixed that 🤷🏻 .